WordPress/Classes/Models/BlogSettings+Discussion.swift

```diff
@@ -0,0 +1,279 @@
+import Foundation
+
+
+/**
+ *  @extension  BlogSettings+Discussion
+ *  @brief      In this extension, we implement several nested Enums (and helper setters / getters)
+ *              aimed at simplifying the BlogSettings interface.
+ *              This may be considered as an Adapter class, *Swift* style!
+ */
+
+extension BlogSettings
+{
+    /**
+     *  @enum   BlogSettings.CommentsAutoapproval
+     *  @brief  Enumerates all of the Comments AutoApproval settings
+     */
+    public enum CommentsAutoapproval : Int {
```

Comment:
Just wondering why `public`? (internal should be fine for the App classes afaik, and for unit testing you can use `@testable import`)